queuebot/cogs/queue/suggestion.py

```diff
@@ -63,11 +64,19 @@ def status(self):
         else:
             status = 'In the private council queue'
 
+        submission_time = self.record["submission_time"] or "Unknown time"
+
+        if self.record["forced_by"]:
+            force_text = f"Forced by: <@{self.record['forced_by']}>\nReason: {self.record['forced_reason']}"
+        else:
+            force_text = ""
+
         return f"""**Suggestion #{self.record['idx']}**
 
-Submitted by <@{self.record['user_id']}>
+Submitted by <@{self.record['user_id']}> at {submission_time}
```

Comment:
How is the formatting on this? Should we include both 12-hour and 24-hour times?

Reply:
It's default datetime `__str__`, so this will always be a 24 hour UTC date